app/components/snippets/box_component.rb

```diff
@@ -11,6 +11,7 @@ def initialize(snippet:, user:)
       @snippet = snippet
 
       @user_can_edit_snippet = @snippet.user == @user && Time.now.utc < Aoc.release_time(@snippet.day) + 48.hours
+      @user_can_discuss_snippet = @snippet.user != @user && @snippet.user.slack_id.present?
```

Comment:
Use the `slack_linked` scope here, I'm going to update (fix) the scope in [this PR](https://github.com/pil0u/lewagon-aoc/pull/697).